src/Elliptic/DiscontinuousGalerkin/NumericalFluxes/FirstOrderInternalPenalty.hpp

```diff
@@ -168,8 +171,9 @@ struct FirstOrderInternalPenalty<Dim, FluxesComputerTag,
     // possible to communicate non-tensors (the element size and the number of
     // collocation points on either side of the mortar), this option will be
     // changed to be just the free parameter `C` multiplying `N_points^2 / h`.
-    // This will improve conditioning on non-uniform meshes. Currently, the
-    // `packaged_data` is a `Variables` which can only hold tensors.
+    // This will improve conditioning on non-uniform meshes. It can be
+    // implemented as of https://github.com/sxs-collaboration/spectre/pull/1725
+    // (and will be ASAP) by using the `package_extra_tags` below.
```

Comment:
removed the comment. I don't need an issue since this is my own todo, will implement this ASAP after this PR is merged (it's one of the features this PR makes possible).